deploying/deploying.md

```diff
@@ -171,7 +171,7 @@ Ensure you don't have any missing files: `git status` and commit them if you nee
 13. Deploy your app!
     `git push heroku master`
     This takes a long time.
-    This will print the URL your app was deployed to. Trying going to it! If something goes run, use `heroku logs --tail` to debug.
+    This will print the URL your app was deployed to. Try going to it! If something goes run, use `heroku logs --tail` to debug.
```

Comment:
```suggestion
    This will print the URL your app was deployed to. Try going to it! If something goes wrong, use `heroku logs --tail` to debug.
```